exchanges/cryptodotcom/cryptodotcom_types.go

```diff
@@ -320,6 +300,85 @@ type CreateOrderParam struct {
 	TriggerPrice   float64    `json:"trigger_price"`
 }
 
+func (arg *CreateOrderParam) getCreateParamMap() (map[string]interface{}, error) {
```

Comment:
This needs full test coverage and utilizing `if !errors.Is(err, errGlobalError) { }`